src/verdant/panel/events/event-date.tsx

```diff
@@ -25,6 +25,25 @@ type NotebookDate_Props = {
 };
 
 class NotebookEventDate extends React.Component<NotebookDate_Props> {
+  // Helper method to conditionally construct the list of events under a date
+  makeEventList = () => {
```

Comment:
(Minor) You don't need the = () => notation on a class function. Just makeEventList() {} will do fine. The () => .... notation is for anonymous functions or assigning functions to variables.